train.py

```diff
@@ -330,8 +329,25 @@ def buildCallbacks(params, model, dataset):
 def main():
     args = parse_args()
     print(args)
-    parameters = load_parameters()
-    if args.config is not None:
+    # load the default config parameters
+    # load the user config and overwrite any defaults
+    if args.config.endswith('.yml'):
+        with open('configs/default-config-BiRNN.yml') as file: #FIXME make this a user option (maybe depend on model type and level?)
+            parameters = yaml.load(file, Loader=yaml.FullLoader)
+        with open(args.config.strip()) as file:
```

Comment:
The `.strip()` is weird. And potentially annoying if for some even weirder reason you actually wanted to create a config file with a name that begins with space.